lib/uk_nationals_living_in_the_eu.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
I think this file may need re-naming to match the path.
 The controller loads it as:
``` 
def qa_config
    @qa_config ||= YAML.load_file("lib/#{request.path.tr('-', '_')}.yaml")
  end
```